model_compression_toolkit/core/pytorch/back2framework/mixed_precision_model_builder.py

```diff
@@ -163,6 +163,12 @@ def mixed_precision_activation_holder(self, n: BaseNode) -> PytorchActivationQua
         activation_conf_nodes_names = [n.name for n in self.graph.get_activation_configurable_nodes()]
 
         activation_quantizers = []
+
```

Comment:
you don't need to modify the mixed precision builder, you can remove all changes from this file